<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exif</class>
 <widget class="QWidget" name="Exif">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="toolTip">
      <string>Example: yyyyMMdd_hhmmss_</string>
     </property>
     <property name="text">
      <string>yyyyMMdd_hhmmss_</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="prependRadio">
     <property name="text">
      <string>prepend</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="appendRadio">
     <property name="text">
      <string>append</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="nothingRadio">
     <property name="text">
      <string>change nothing</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
